src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -295,18 +295,21 @@ def read_geo_file(self,):
                 raise(RuntimeError("No supplied levelpool parameters in routing config"))
                 
             lake_id = levelpool_params.get("level_pool_waterbody_id", "wb-id")
-            self._waterbody_df = read_ngen_waterbody_df(
-                        levelpool_params["level_pool_waterbody_parameter_file_path"],
-                        lake_id,
-                        )
-                
-            # Remove duplicate lake_ids and rows
-            self._waterbody_df = (
-                            self.waterbody_dataframe.reset_index()
-                            .drop_duplicates(subset=lake_id)
-                            .set_index(lake_id)
-                            .sort_index()
+            try:
+                self._waterbody_df = read_ngen_waterbody_df(
+                            levelpool_params["level_pool_waterbody_parameter_file_path"],
+                            lake_id,
                             )
+                    
+                # Remove duplicate lake_ids and rows
+                self._waterbody_df = (
+                                self.waterbody_dataframe.reset_index()
+                                .drop_duplicates(subset=lake_id)
+                                .set_index(lake_id)
+                                .sort_index()
+                                )
+            except ValueError:
+                self._waterbody_df = pd.DataFrame()
```

Comment:
I think this is reasonable for now, but might be a good idea to log some details later, especially if someone may be expecting to have reservoir data, but it doesn't read correct or is missing...this silent failure might be difficult to track down.

Reply:
I agree. I need to go back and update a lot of the logging/debugging as much of it got jumbled up when we transferred to HYFeatures. I'll add this to that list.